config/stripe/plans.rb

```diff
@@ -0,0 +1,31 @@
+# This file contains descriptions of all your stripe plans
+
+# Example
+# Stripe::Plans::PRIMO #=> 'primo'
+
+Stripe.plan :DreamTimePlan do |plan|
+  # plan name as it will appear on credit card statements
```

Comment:
Maybe don't need the out-of-the-box description comments here now that the `plan` method is in place?

Reply:
That is a very valid point, but we believe that there is still important information in the comments that could help us when changing e.g rates.

Reply:
Fair enough

Reply:
I agree with William, you should clean this file from the generated comments. If you need to find something out from them- we have version control and you can always go back and look!

Reply:
Okidok, the comments have been removed